frontend/tsconfig.json

```diff
@@ -18,7 +18,19 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+
+    /* Absolute imports */
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["src/components/*"],
+      "@/consts/*": ["src/consts/*"],
+      "@/routes/*": ["src/routes/*"],
+      "@/services/*": ["src/services/*"],
+      "@/slices/*": ["src/slices/*"],
+      "@/types/*": ["src/types/*"],
+      "@/view/*": ["src/view/*"]
+    }
```

Comment:
Just a comment again: thank you for having taken into account my comments from previous pull requests 🤩